synapse/rest/client/room.py

```diff
@@ -814,11 +814,17 @@ async def on_GET(
                 [event], requester.user.to_string()
             )
 
+            serializer_options = SerializeEventConfig(requester=requester)
+
             time_now = self.clock.time_msec()
             # per MSC2676, /rooms/{roomId}/event/{eventId}, should return the
             # *original* event, rather than the edited version
             event_dict = self._event_serializer.serialize_event(
-                event, time_now, bundle_aggregations=aggregations, apply_edits=False
+                event,
+                time_now,
+                bundle_aggregations=aggregations,
+                apply_edits=False,
+                config=serializer_options,
```

Comment:
Again I think it might be OK to inline this?

Reply:
Will do